mosaic-site/src/_index/features/Features.module.css

```diff
@@ -1,3 +1,7 @@
+.listItem > svg {
+  --saltIcon-color: var(--salt-actionable-primary-background-active);
```

Comment:
Just checking, shouldn't this be 'foreground'?
CC @libertymayc

Reply:
could be the right choice, was just checking. If I remember well, @pseys mentioned we should use foreground for SVGs, but he might be referring only to Salt components